cmd/worker/build.sh

```diff
@@ -12,7 +12,7 @@ cleanup() {
 trap cleanup EXIT
 
 
-if [[ "${DOCKER_BAZEL:-false}" == "true" ]]; then
+if [[ "$DOCKER_BAZEL" == "true" ]]; then
```

Comment:
This should not be reverted. Same with the other ones
```suggestion
if [[ "${DOCKER_BAZEL:-false}" == "true" ]]; then
```